recipes/head_node_slurm_finalize.rb

```diff
@@ -49,14 +49,14 @@
     require 'shellwords'
 
     # Example output for sinfo
-    # $ /opt/slurm/bin/sinfo -N -h -o '%N %t'
+    # $ #{node['cfncluster']['slurm']['install_dir']}/bin/sinfo -N -h -o '%N %t'
```

Comment:
minor: I think we don't need to change it in the comment. Because in the comment we are running linux command, which is not related to chef node attribute.